File: .vscode/.py/[#8].py
'''[#8] Leia dois ńúmeros e que pergunte qual operação você deseja realizar. Você deve poder calcular soma 
(+), subtração (-), multiplicação (*) e divisão (/). Exiba o resultado da operação solicitada.'''
import os

num1 = float(input("Digite um primeiro número: "))
num2 = float(input("Digite outro número: "))

print("\nEscolha uma das opções abaixo, digitando o número da operação:")
esc = int(input("1 - Soma (+)\n2 - Subtração (-)\n3 - Multiplicação (*)\n4 - Divisão (/)\n"))

os.system ("cls")

if (esc == 1):
    print(f"Seu resultado é:", num1 + num2)
elif (esc == 2):
    print(f"Seu resultado é:", num1 - num2)
elif (esc == 3):
    print(f"Seu resultado é:", num1 * num2)
elif (esc == 4 and num2 != 0):
    print(f"Seu resultado é:", num1 / num2)
elif (esc == 4 and num2 == 0):
    print("ERR.")
else:
    print("ERR.")

File: .vscode/.c-.c++/[#8].c
#include <stdio.h>
int main(){

    float num1, num2;
    int esc;

    printf("Escolha um número: ");
    scanf("%f", &num1);
    printf("Escolha outro número: ");
    scanf("%f", &num2);

    printf("\nEscolha uma das opções abaixo, digitando o número da operação: ");
    printf("\n1 - Soma (+)\n2 - Subtração (-)\n3 - Multiplicação (*)\n4 - Subtração (/)\n");
    scanf("%d", &esc);

    if (esc == 1){
        printf("\nO resultado é:%.2f", num1 + num2);
    }
    else if (esc == 2){
        printf("\nO resultado é:%2.f", num1 - num2);
    }
    else if (esc == 3){
        printf("\nO resultado é:%2.f", num1 * num2);
    }
    else if (esc == 4 && num2 != 0){
        printf("\nO resultado é:%2.f", num1 / num2);
    }
    else if (esc == 4 && num2 == 0){
        printf("\nERR.");
    }
    else{
        printf("\nERR.");
    }
    return 0;
}